### ISSUE TITLE
Default property alias

### ISSUE THREAD

**Issue by machtelik** [Quick-Turn-Studio/CLionSupportForQt#15]
The following code leads to error `QmlTokenType.CLASS_NAME, QmlTokenType.IMPORT_QUALIFIER, QmlTokenType.KEYWORD_TYPE or QmlTokenType.LIST_KEYWORD expected, got 'alias'`

```
    default property alias children: customContent.children
```

**Comment by grabusr** [comment]
Hi Mike

Version 0.9.3 with a fix for that is available to download. Please confirm if it solves the issue.

**Comment by machtelik** [comment]
It's fixed in 0.9.3, thanks.
